tests/functionTests/completionTests_fish.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 IBM Corporation
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Structure Test: 1st line cmd input, Starting form 2nd line: Expected suggestions
+
+
+##############################################################################
+# Complex type RPCs
+##############################################################################
+
+#START_TEST recursive datastructures number_and_string
+@@CMD@@ --complete=fish 0.0.0.0:50051 examples.NestedTypeRpcs echoNestedMaps number_and_string=:number=5
```

Comment:
test looks good :smile:

Reply:
maybe add some more examples with different scenarios, this will help designing a robust algorithm :)